### DISCUSSION TITLE
Design Website UI/UX #12

### DISCUSSION THREAD

**Discussion by Samsonroyal** [oscakampala/oscakampala.github.io#13] [Ideas]
Let's use this as an avenue to brainstorm about what we need to add or remove from the features in the figma files designed.

#### Useful Links
[figma file](https://www.figma.com/file/uuCKSWJrYAKRjUuIMTWKTv/OSCA-Kampala-Website-team-library?node-id=429%3A15)
[Github Issue](https://github.com/oscakampala/oscakampala.github.io/issues/12)
[Kanban board](https://github.com/oscakampala/oscakampala.github.io/projects/1)

**Comment by Samsonroyal** [comment]
The checklist that needs to be cleared for the homepage


- [ ] Description section for running open-source projects in the community

 

- [ ] Blogging section

 

- [ ] Team Leads page

 

- [ ] Future events

 

- [ ] Newsletter Subscription section(this can also be a pop-up)




#### These can be Entirely dedicated pages
 

- [x] Dedicated Summit Page(Sponsors, Powerpoints, Talks, Agenda, Donation options) , this is a separate website link

- [ ] E-commerce page for merchandise

- [ ]  An open-source chat section such as a ranking system for most contributors

- [ ] Download page for fully-scaled out applications

@KakoozaJerry @Sheba-Ayinzabyonna @ht-thomas @PiusKevin3 @Lincoln-developer @SyfaCollins

**Reply by TobiasHT5** [reply]
Yup... this is a comprehensive checklist. Thanks for taking the time to compile it. The web team should begin working on this ASAP.

**Reply by Sheba-Ayinzabyonna** [reply]
E-commerce page for the products for sale.

**Reply by KakoozaJerry** [reply]
Blog section should point to [https://blog.oscakampala.org/](https://blog.oscakampala.org/) , no need to build up CRUD functionality since we are leveraging hashnode's power

**Reply by Samsonroyal** [reply]
I like the simplification of work

**Comment by SyfaCollins** [comment]
![caseshow](https://user-images.githubusercontent.com/66389234/173448187-ac7b6df2-b990-40f7-837c-3279e682d280.png)
Hey, y'll hope we good. I was able to accomplish some of the Web page  designs check out [here](https://www.figma.com/file/uuCKSWJrYAKRjUuIMTWKTv/OSCA-Kampala-Website-team-library?node-id=424%3A76),
I'd say its high time we reviewed our design and got started with some codes, as we shall be adding more feature designs
bit per bit.

**Reply by Samsonroyal** [reply]
Hello, @SyfaCollins awesome work you've executed. I checked out the Figma file and I have to say it's stunning. 

**Suggestions**: 

1. The button for "join the community" can the words be shortened a bit. To something like (Join Us, Join community or Join) I fear the button is too long. 
2. And I don't know if it's just me but the color used on the buttons in the shop/catalog page are hurting to the eyes(they are fighting with the white text.) probably consider choosing a darker background to balance the contrast.

**Reply by SyfaCollins** [reply]
Thanks, @Samsonroyal.
So I think we shall go with "Join Community"  and a darker bg-color for those buttons as you suggested.

**Comment by SyfaCollins** [comment]
Good afternoon, folks. Had to make an announcement 4 days back after pushing some code to our project.
I was able to implement some of our designs from Figma and @PiusKevin3 merged them. So you can checkout
https://oscakampala.org, its incomplete so anyone can pull and topup some code.
I included [tailwind CSS](https://tailwindcss.com/docs/installation) and [react-icons](https://react-icons.github.io/react-icons/)

**Reply by Samsonroyal** [reply]
Wow @SyfaCollins  this is pristine; it is beautiful, the immaculateness of this website can only be surpassed by the holy grail of code. Heavens gracious.

**Comment by Lincoln-developer** [comment]
Thanks, it's really really great! That we are having things like the
merchandise section, have liked the whole projection
